command/sub8_missions/missions/collect_classify.py

```diff
@@ -320,11 +328,11 @@ def drop_pipe(self, pose):
         # Move at safe depth above oval
         position = pose[0]
         position = position + CollectClassifyMission.BASE_LINK_TO_ARM
-        position[2] = CollectClassifyMission.OVAL_SEARCH_DEPTH
+        position[2] = -self.height_to_depth(CollectClassifyMission.OVAL_SEARCH_HEIGHT)
         yield self.sub.move.set_position(position).set_orientation(pose[1]).zero_roll_and_pitch()
```

Comment:
Similarly you could add `.to_height(...)` at the end of these as well. It would make it more obvious what the intent was, though it would make the line a lot longer.